### PR TITLE
fix: [CDS-86829]: Fix Styling - file path should be wrapped. its goin…

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
Before

Before

After

Screenshot 2023-12-14 at 2 44 01 PM

Screenshot 2023-12-14 at 2 44 04 PM

Tests
- tested with small test
- tested with big text - it shows truncated and shows tooltip

### CODE REVIEW

File: src/modules/70-pipeline/components/ManifestSelection/__tests__/__snapshots__/ManifestSelection.test.tsx.snap [hunk 1]
```diff
@@ -990,6 +991,7 @@ exports[`ManifestSelection tests renders add Manifest option without crashing 1`
                       <div
                         class="StyledProps--font StyledProps--main StyledProps--flex StyledProps--flex-justifyContent-flex-start"
                         data-testid="1"
+                        style="flex: 1;"
```
Comment (vikrant-harness) [inline src/modules/70-pipeline/components/ManifestSelection/__tests__/__snapshots__/ManifestSelection.test.tsx.snap:994]: @ishantmehtaharness let's remove the snapshot testing with the same PR ?

Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/ManifestSelection/__tests__/__snapshots__/ManifestSelection.test.tsx.snap:994]: Removing will reduce the coverage

Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/ManifestSelection/__tests__/__snapshots__/ManifestSelection.test.tsx.snap:994]: And I see this file only has snapshots. We should plan to refactor that and remove snapshots

Comment (vikrant-harness) [reply src/modules/70-pipeline/components/ManifestSelection/__tests__/__snapshots__/ManifestSelection.test.tsx.snap:994]: Yes we need to remove snapshots and write proper assertions even if it reduces coverage initially, but sure you can create one ticket and tag it here Thanks! CC :- @chetanvekariya-harness

**Review by vikrant-harness — APPROVED** [review]

**Review by satyam-harness — APPROVED** [review]

**Pull request merged**